physical/zookeeper.go

```diff
@@ -0,0 +1,213 @@
+package physical
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/armon/go-metrics"
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+// ZookeeperBackend is a physical backend that stores data at specific
+// prefix within Zookeeper. It is used in production situations as
+// it allows Vault to run on multiple machines in a highly-available manner.
+type ZookeeperBackend struct {
+	path   string
+	client *zk.Conn
+}
+
+// newZookeeperBackend constructs a Zookeeper backend using the given API client
+// and the prefix in the KV store.
+func newZookeeperBackend(conf map[string]string) (Backend, error) {
+	// Get the path in Zookeeper
+	path, ok := conf["path"]
+	if !ok {
+		path = "vault/"
+	}
+
+	// Ensure path is suffixed but not prefixed
```

Comment:
Thanks! Yes, i just didnt update the comment from the copy and paste from the consul code. Updated.